easybuild/easyconfigs/c/canu/canu-1.9-GCCcore-8.3.0-Perl-5.30.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'canu'
+version = '1.9'
+versionsuffix = '-Perl-%(perlver)s'
```

Comment:
I think we can/should drop the Perl suffix; there is just one perl version per toolchain release (this seems to be the case in the packages for GCCcore 8.3.0 so far). However; Java can and does vary, so '-Java-%(javaver)s' instead.

Reply:
+1 on this

Reply:
done